workbench-core/example/app/src/routes/dynamicAuthorizationRoutes.ts

```diff
@@ -106,6 +106,28 @@ export function setUpDynamicAuthorizationRoutes(router: Router, service: Dynamic
     })
   );
 
+  router.delete(
+    '/authorization/groups/:groupId',
+    wrapAsync(async (req: Request, res: Response) => {
+      try {
+        await service.deleteGroup({
+          groupId: req.params.groupId,
+          authenticatedUser: res.locals.user
+        });
+
+        res.status(204).send();
```

Comment:
lets return the `data` response object that includes the deleted group.

Reply:
Isn't that redundant? `groupId` is already in URL path and standard DELETE verb usually returns 204 if resource was successfully deleted.

Reply:
its useful to return the deleted resource so the caller can use it if need be, it also matches or response from the service for consistency